client/blocks/site/style.scss

```diff
@@ -118,6 +119,39 @@
 	}
 }
 
+.site__link-hint {
+	flex: 0 1 auto;
+	position: relative;
+
+	// from layout/sidebar/styles.scss a.sidebar-button
+	box-sizing: border-box;
+	white-space: nowrap;
+	overflow: hidden;
+	padding: 2px 8px 3px 8px;
+	height: 24px;
+	margin: 11px 8px 0 0;
+	line-height: 18px;
+	background-color: $gray-light;
+	color: darken( $gray, 20% );
+	font-size: 11px;
+	font-weight: 600;
+	border-radius: 3px;
+	border: 1px solid darken( $sidebar-bg-color, 10% );
```

Comment:
Shouldn't use sidebar colors for this item because it can live outside of the sidebar.